### DISCUSSION TITLE
Question: How are you working with the variables as a value?

### DISCUSSION THREAD

**Discussion by RVledder** [six7/figma-tokens#169] [Q&A]
For the typography the following is happening: 

**./design-tokens.json**
```
{
  "fontFamilies": {
    "font-arial": "Arial"
  },
  "typography": {
    "headers": {
      "fontFamily": "$fontFamilies.font-arial"
    }
  }
}
```
How are you using that in your application?

Typically json-sass ([example](json-sass)) tools don't handle variables to generate a value do they?

**Comment by six7** [comment]
Indeed, variables is not json-conform. The JSON tab in the plugin is more like an "input" than an Export tab. When we finally include a method to Export to valid JSON, all these variables should be converted to their real values.

**Comment by nathancooper** [comment]
Regarding variables as a value, can you do calculations with them? Thank you!

Example:

```
{
    "sizing": {
        "usr-base": "8px",
        "usr-base-2x": "$sizing.usr-base * 2"
    }
}
```

**Comment by six7** [comment]
> Regarding variables as a value, can you do calculations with them? Thank you!
> 
> Example:
> 
> ```
> {
>     "sizing": {
>         "usr-base": "8px",
>         "usr-base-2x": "$sizing.usr-base * 2"
>     }
> }
> ```

Totally missed your reply @nathancooper — yes you can! What you wrote should work.
